airbyte-integrations/connectors/source-posthog/unit_tests/test_components.py

```diff
@@ -85,42 +86,48 @@ def test_update_cursor(test_name, initial_state, stream_slice, last_record, expe
     [
         ("test_empty_state",
          {},
-         [{'end_time': '2021-01-10T00:00:00.000000+0000',
+         [{'end_time': '2021-01-10T23:59:59.999999+0000',
            'project_id': '2331',
            'start_time': '2021-01-01T00:00:00.000000+0000'},
```

Comment:
The start time here would have normally changed since we don't do `state_start_time + 1 day` when we start from the state but the posthog source fix the start time from the first slice: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-posthog/source_posthog/components.py#L118